metaflow/plugins/conda/conda_step_decorator.py

```diff
@@ -81,9 +81,6 @@ def _python_version(self):
         )
 
     def is_enabled(self, ubf_context=None):
```

Comment:
Ditto here.

Reply:
I am again not sure we want to re-enable conda for pure control tasks. The logic there was that conda was defined by the user and may be irrelevant for the non-user control task.

Reply:
@tuulos suggested we enable conda for control tasks as well. @tuulos  do you want to chime in? Personally I feel it would be weird to have frankenstein situation where the control task runs with different libraries than the normal tasks.

Reply:
@romain-intel same deal as in the above comment - the new use cases require `@conda` for control. We can change the behavior for new decorators if needed